### PR TITLE
Replace removed SkinTemplateOutputPageBeforeExec hook

### PR DESCRIPTION
**Pull request by paladox**
Bug: https://phabricator.miraheze.org/T11240

### CODE REVIEW

**Comment by paladox** [comment]
@labster

File: HideSection.hooks.php [hunk 1]
```diff
@@ -59,32 +59,36 @@ public static function onSkinEditSectionLinks( $skin, $title, $section, $tooltip
...
             );
 
+            $title = $out->getPageTitle();
+    		if ( !$title instanceof Title ) {
+    			return;
+    		}
+
             // Append to page title
-            $template->data['title'] .= $hideelem;
+            $out->setPageTitle( $title . ' ' . $hideelem );
```
Comment (labster) [inline HideSection.hooks.php:90]: Did you test this change?  You added whitespace here, which does have layout effects in HTML.

Comment (paladox) [reply HideSection.hooks.php:90]: No, I tried to find an example on how to migrate and found something that looked like it was related.

**Comment by pppery** [comment]
I tested this and it turned out it was broken. Filed a tested and working version as #5

**Comment by labster** [comment]
I merged #5, which should mean this one is no longer necessary.

**Pull request closed without merging**